cinn/hlir/framework/buffer.h

```diff
@@ -50,6 +54,11 @@ struct Buffer final {
     return memory_mng_cache_->malloc(size);
   }
 
+  inline void* Aligned_Alloc(uint32_t alignment, uint32_t size) CINN_RESULT_SHOULD_USE {
```

Comment:
AlignedAlloc ? 

Is the `_` necessary?

Reply:
done,thanks~